packages/pwa-kit-build/scripts/upload.test.js

```diff
@@ -5,6 +5,8 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+require('regenerator-runtime/runtime')
```

Comment:
We probably need to support async/await for all tests.. The way this works will only benefit this file?